splash/resources.py

```diff
@@ -347,6 +347,18 @@ def render_POST(self, request):
         }).encode('utf-8')
 
 
+class PingResource(Resource):
+    isLeaf = True
+
+    def render_GET(self, request):
+        request.setHeader(b"content-type", b"application/json")
+        return json.dumps({
+            "status": "ok",
+            "maxrss": resource.getrusage(resource.RUSAGE_SELF).ru_maxrss,
```

Comment:
This has a problem - on OS X RUSAGE_SELF is in bytes, but on Linux it is in KBs. Check https://github.com/scrapinghub/splash/blob/3c9970ac96096a444ae9eb345d9670adc72bfbef/splash/utils.py#L114.